docs/api-docs/getting-started/best-practices.mdx

```diff
@@ -5,15 +5,15 @@ keywords: rate limits, best practices,
 
 # API Best Practices
 
-## Keep your integration up-to-date
+## Keep your integration up to date
 
-BigCommerce frequently enhances its core product and is actively developing v3 API endpoints. By using the newest API version, you will ensure that your app has access to the latest resources. You will also be better positioned to provide a user experience consistent with what merchants see in their BigCommerce store’s control panel. To stay up to date, bookmark our [changelog](/changelog).
+BigCommerce frequently enhances its core product and is actively enhancing REST endpoints, as well as expanding the graphs accessible to our growing family of GraphQL APIs. Using the latest information lets you update your app to take advantage of the most current resources. You also position your app or implementation to provide a user experience consistent with what merchants see in their BigCommerce store control panel. To stay up to date, bookmark our [changelog](/changelog).
 
 ## Anticipate changes to BigCommerce APIs
 
 At BigCommerce, we make a distinction between "breaking" and "non-breaking" changes.
 
-In most cases, we will give advance notice in our developer [changelog](/changelog) when we make any of the following "breaking" changes. However, we will make breaking changes _without warning_ to non-production (alpha/beta) endpoints, or when analytics indicate that an endpoint has no traffic.
+In most cases, we will give advance notice in our developer [changelog](/changelog) when we make any of the following "breaking" changes. However, we will make breaking changes _without warning_ to alpha and beta endpoints and graph nodes, or when analytics indicate that an endpoint has no traffic.
```

Comment:
Sounds like we should have a specific time period for breaking changes "when analytics indicate that an endpoint has no traffic". I recommend we start at "no traffic for 3 months", as we typically are reviewing platform usage patterns in detail at that interval.

Reply:
v good